lib/local-aspects/iam-path/index.ts

```diff
@@ -0,0 +1,28 @@
+import { IAspect } from "aws-cdk-lib";
+import { IConstruct } from "constructs";
+import * as iam from "aws-cdk-lib/aws-iam";
+
+export class IamPathAspect implements IAspect {
+  private readonly iamPath: string;
+
+  constructor(iamPath: string) {
+    this.iamPath = iamPath;
+  }
+
+  public visit(node: IConstruct): void {
+    if (node instanceof iam.Role) {
```

Comment:
Do you think rewriting these to [Type Guards](https://www.typescriptlang.org/docs/handbook/2/narrowing.html#using-type-predicates) would help to avoid casting?